### PR TITLE
Add a batch of new awesome apps

### PR DESCRIPTION
**Pull request by TongKuo**
For details, please check out [`Commits`](https://github.com/iCHAIT/awesome-osx/pull/60/commits).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,81 +115,58 @@
...
 
-
 ## License
-
 [![Creative Commons License](http://i.creativecommons.org/l/by/4.0/88x31.png)](http://creativecommons.org/licenses/by/4.0/)
 
 This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
 
-[OSS Icon]: media/oss.png
+[oss icon]: media/oss.png
```
Comment (iCHAIT) [inline README.md]: I think you have broken OSS image.

File: README.md [hunk 2]
```diff
@@ -125,81 +115,58 @@
...
-* https://gist.github.com/kevinelliott/3135044
-* https://gist.github.com/kimmobrunfeldt/350f4898d1b82cf10bce
+- [https://gist.github.com/kevinelliott/3135044](https://gist.github.com/kevinelliott/3135044)
+- [https://gist.github.com/kimmobrunfeldt/350f4898d1b82cf10bce](https://gist.github.com/kimmobrunfeldt/350f4898d1b82cf10bce)
 
 ### OS X 10.10 Yosemite Setup
-
-https://gist.github.com/kevinelliott/0726211d17020a6abc1f
-
+[https://gist.github.com/kevinelliott/0726211d17020a6abc1f](https://gist.github.com/kevinelliott/0726211d17020a6abc1f)
```
Comment (iCHAIT) [inline README.md]: Just out of curiosity what difference does it makes from the earlier one?

File: README.md [hunk 3]
```diff
@@ -82,40 +78,34 @@
...
 - [Helium](http://heliumfloats.com/) - A floating browser window that allows you to watch media while you work. ![Open-Source Software][OSS Icon]
 - [iStat Menus](https://bjango.com/mac/istatmenus/) - An advanced system monitor for your menubar.
 - [Kawa](https://github.com/noraesae/kawa) - A better input source switcher with shortcuts. ![Open-Source Software][OSS Icon]
 - [Keka](http://www.kekaosx.com/en/) - Compress to and extract from many archive file formats. ![Open-Source Software][OSS Icon]
+- [Mactracker](http://www.mactracker.ca) - A tool provides detailed information on every Apple Macintosh computer ever made, including items such as processor speed, memory, optical drives, graphic cards, supported OS versions, and expansion options. Also included is information on early Apple systems, Apple mice, keyboards, displays, printers, scanners, speakers, cameras, iPod, Apple TV, iPhone, iPad, Apple Watch, Wi-Fi products, Newton, iOS, Mac OS, OS X, and watchOS versions.
 - [Mackup](https://github.com/lra/mackup) - Keep your application settings in sync. ![Open-Source Software][OSS Icon]
```
Comment (iCHAIT) [inline README.md]: No need to describe it in detail,  `A tool provides detailed information on every Apple Macintosh computer ever made,` should be fine.

File: README.md [hunk 4]
```diff
@@ -46,33 +38,37 @@
...
 
+### Design
+- [Sketch](http://bohemiancoding.com/sketch/) - An awesome vector editor like Adobe Illustrator. Sketch gives you the power, flexibility and speed you always wanted in a lightweight and easy-to-use package. Finally you can focus on what you do best: Design.
+- [PaintCode](http://www.paintcodeapp.com) - A powerful tool to turn your vector drawings into code.
 
 ### Editors
-
 - [Atom](https://atom.io/) - The hackable text editor. ![Open-Source Software][OSS Icon]
 - [Brackets](http://brackets.io/) - A modern text editor that understands web design. ![Open-Source Software][OSS Icon]
+- [CotEditor](http://coteditor.com) - Lightweight Plain-Text Editor for OS X. ![Open-Source Software][OSS Icon]
```
Comment (iCHAIT) [inline README.md]: Description should be `Lightweight Plain-Text Editor` not need to use `for OS X`

File: README.md [hunk 5]
```diff
@@ -46,33 +38,37 @@
 - [WWDC](https://github.com/insidegui/WWDC) - The WWDC app. ![Open-Source Software][OSS Icon]
 - [Zsh](http://www.zsh.org/) -  A powerful command line shell. ![Open-Source Software][OSS Icon]
 
+### Design
+- [Sketch](http://bohemiancoding.com/sketch/) - An awesome vector editor like Adobe Illustrator. Sketch gives you the power, flexibility and speed you always wanted in a lightweight and easy-to-use package. Finally you can focus on what you do best: Design.
+- [PaintCode](http://www.paintcodeapp.com) - A powerful tool to turn your vector drawings into code.
```
Comment (iCHAIT) [inline README.md]: Again the description should not be that big, you should try and sum it up in 1 line, maybe something like `A Professional Digital Design Editor`?

File: README.md [hunk 6]
```diff
@@ -46,33 +38,37 @@
...
-
 ### Games
 - [OpenEmu](http://openemu.org/) - Multiple Video Game System. ![Open-Source Software][OSS Icon]
 - [Screentendo](https://github.com/AaronRandall/Screentendo) - Turn your screen into a playable level of Mario. ![Open-Source Software][OSS Icon]
 - [stockfish-mac](https://github.com/daylen/stockfish-mac) - Beautiful, powerful chess application. ![Open-Source Software][OSS Icon]
 
+### Movies
+- [Popcorn Time](https://popcorntime.io) - An app to watch Movies and TV Shows instantly. ![Open-Source Software][OSS Icon]
+- [Transmission](http://www.transmissionbt.com) - An open source, volunteer-based project. Unlike some BitTorrent clients, Transmission doesn't play games with its users to make money. ![Open-Source Software][OSS Icon]
+- [VLC](http://www.videolan.org/vlc/index.html) - Simple, fast and powerful media player that has a native UI. ![Open-Source Software][OSS Icon]
```
Comment (iCHAIT) [inline README.md]: I don't think VLC fits into this list, though it is an awesome product but still it is a very trivial product and very well known.

Comment (iCHAIT) [inline README.md]: I think the description should be `A fast, easy and free BitTorrent Client.``

**Comment by iCHAIT** [comment]
Hi, I have left some comments that I feel need changes, other than that I think you have made some nice additions.

BTW, I don't like the fact that you have removed newlines from the list, though it would not affect the rendered README significantly but would definitely make the raw less readable.

**Comment by TongKuo** [comment]
Okay, I'll fix them ASAP.

**Comment by TongKuo** [comment]
How about now? If I forgot anything just tell me.

**Comment by TongKuo** [comment]
> BTW, I don't like the fact that you have removed newlines from the list, though it would not affect the 
> rendered README significantly but would definitely make the raw less readable.

Sorry, I have never removed them explicitly, I promise. It may be modified by Atom's auto-format functionality. Anyway, I changed them back.

File: README.md [hunk 7]
```diff
@@ -1,8 +1,7 @@
 # Awesome OS X [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
-
 > A curated list of awesome applications, softwares, tools and shiny things for OS X.
 
-> *Items marked with ![Open-Source Software][OSS Icon] are open-source software.*
+> _Items marked with ![Open-Source Software][OSS Icon] are open-source software._
```
Comment (iCHAIT) [inline README.md]: No need to change this, it was fine as it was.

File: README.md [hunk 8]
```diff
@@ -14,29 +13,26 @@
...
-
 ### Chat Clients
 
-
 - [ChitChat](https://github.com/stonesam92/ChitChat) - A native Mac app wrapper for WhatsApp Web. ![Open-Source Software][OSS Icon]
 - [LimeChat](http://limechat.net/mac/) - Instant messaging application. ![Open-Source Software][OSS Icon]
 - [Messenger for Desktop](http://messengerfordesktop.com/) - An app for Facebook messenger. ![Open-Source Software][OSS Icon]
 - [Textual 5](https://www.codeux.com/textual/) - An Internet Relay Chat (IRC) client. ![Open-Source Software][OSS Icon]
 
-
```
Comment (iCHAIT) [inline README.md]: The basic trend is to leave 2 newlines before starting a new category or sub category to make it more readable, you should stick to that.

File: README.md [hunk 9]
```diff
@@ -46,30 +42,39 @@
...
 - [Brackets](http://brackets.io/) - A modern text editor that understands web design. ![Open-Source Software][OSS Icon]
+- [CotEditor](http://coteditor.com) - Lightweight Plain-Text Editor. ![Open-Source Software][OSS Icon]
 - [Light Table](https://github.com/LightTable/LightTable) - A customizable editor with REPL support for Clojure, JavaScript, Python and others. ![Open-Source Software][OSS Icon]
 - [Macvim](https://github.com/b4winckler/macvim) - Vim, the text editor. ![Open-Source Software][OSS Icon]
+- [Mou](http://25.io/mou/) A Lightweight Markdown editor.
 - [Sublime Text 3](http://www.sublimetext.com/) - The sophisticated text editor.
 - [Textmate](https://macromates.com/) - A graphical text editor. ![Open-Source Software][OSS Icon]
 - [Visual Studio Code](https://code.visualstudio.com/) - Build and debug modern web and cloud applications.
 
-
```
Comment (iCHAIT) [inline README.md]: 2 newlines!

File: README.md [hunk 10]
```diff
@@ -46,30 +42,39 @@
...
 ### Finder
 
 - [Path Finder](http://www.cocoatech.com/pathfinder/) - A powerful dual-pane browser alternative to Finder.
 - [TotalFinder](http://totalfinder.binaryage.com/) - A powerful alternative to Finder.
 - [XtraFinder](https://www.trankynam.com/xtrafinder/) - Adds useful features to Finder.
 
-
```
Comment (iCHAIT) [inline README.md]: here also.

File: README.md [hunk 11]
```diff
@@ -133,61 +135,53 @@
...
 
-* [MIDAS](https://github.com/etsy/MIDAS) - Intrusion Detection Analysis System.
-* [OS-X-Yosemite-Security-and-Privacy-Guide](https://github.com/drduh/OS-X-Yosemite-Security-and-Privacy-Guide)
-* [OSXAuditor](https://github.com/jipegit/OSXAuditor) - A free computer forensics tool.
-* [santa](https://github.com/google/santa) - A binary whitelisting/blacklisting system.
-
+- [MIDAS](https://github.com/etsy/MIDAS) - Intrusion Detection Analysis System.
+- [OS-X-Yosemite-Security-and-Privacy-Guide](https://github.com/drduh/OS-X-Yosemite-Security-and-Privacy-Guide)
+- [OSXAuditor](https://github.com/jipegit/OSXAuditor) - A free computer forensics tool.
+- [santa](https://github.com/google/santa) - A binary whitelisting/blacklisting system.
```
Comment (iCHAIT) [inline README.md]: This should not be changed.

File: README.md [hunk 12]
```diff
@@ -195,11 +189,10 @@ Contributions are most welcome, try to adhere to the [contribution guidelines](c
...
 
-
 ## License
 
 [![Creative Commons License](http://i.creativecommons.org/l/by/4.0/88x31.png)](http://creativecommons.org/licenses/by/4.0/)
 
 This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
 
-[OSS Icon]: media/oss.png
+[OSS icon]: media/oss.png
```
Comment (iCHAIT) [inline README.md]: The OSS logo is still broken, it should be - `[OSS Icon]: media/oss.png`

**Comment by iCHAIT** [comment]
Left some comments, apart from that you need to resolve conflicts.

File: README.md (hunk 6 above)
Comment (herrbischoff) [inline README.md]: I'm skeptical about whether

- **Popcorn Time** should be on a list like this at all. Its sole function is to pirate movies, which is forbidden by law in most parts of the world. Besides that, it has no legal application scenario whatsoever. Whether this is felt to be right or wrong is a matter of personal opinion but for now its use is still considered unlawful.
- **Transmission** should be in a "Movies" category. It's better suited under "Sharing files".

Comment (iCHAIT) [inline README.md]: Totally agree with @herrbischoff

**Comment by herrbischoff** [comment]
@TongKuo: Are you going to clean up your pull request or have you possibly lost interest? If so, please let us know so others can add some applications in your PR. As it stands, those would be considered duplicates. Thanks.

**Comment by TongKuo** [comment]
Nope, I didn't lose interest, I'll fix them ASAP, sorry for delay.

**Comment by TongKuo** [comment]
I'll re-submit a new p-r.

**Comment by herrbischoff** [comment]
Good to hear! Please submit one pull request per application instead of one humongous from now on. This makes them far easier to merge.

Also, descriptions should be factual and as neutral as possible. Although it can be found on its website, please refrain from largely unspecific and opinion-based descriptions like:

> An open source, volunteer-based project. Unlike some BitTorrent clients, Transmission doesn't play games with its users to make money.

A better description would be:

> A fast, easy and free BitTorrent Client.

**Comment by TongKuo** [comment]
Okay, no problem.

**Pull request closed without merging**